mitreattack/navlayers/exporters/svg_objects.py

```diff
@@ -422,8 +425,15 @@ def _com_color(self, technique, tactic, colors=[]):
             :return: Hex color code
         """
         c = 'FFFFFF'
-        if technique.score:
-            c = self.grade.compute_color(technique.score)[1:]
+        tscore = None
+        if technique.score is not None:
+            tscore = technique.score
+        try:
+            tscore = technique.aggregateScore
+        except AttributeError:
+            pass
```

Comment:
Surely there must be a better way to check if the technique has an aggregate score than try-catch? If this is the best way to do it it's acceptable, it just seems like using try/catch for branching logic is bad practice.

Reply:
I'll switch it to a slightly different approach.